CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 # Braintree iOS SDK Release Notes
 
-## unreleased
+## unreleased (next-major-version)
 * Breaking Changes
   * Bump minimum supported deployment target to iOS 13+
   * Require Xcode 13
     * Bump Swift Tools Version to 5.5 for CocoaPods & SPM
+  * BraintreeVenmo
+    * Remove `.unspecified` case from `BTVenmoPaymentMethodUsage` enum
```

Comment:
Do we want to note the enum number change here too?

Reply:
Hm - I don't think we need to call it out. Obj-C devs should know about how enums use the first value as the default (or zero) case. If you think it is confusing though, we can call it out explicitily!

Reply:
Ah no I forgot we already discussed that 👍